CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+- If anything isn't clear in the code, feel free to create an
+  [issue on GitHub][GitHubIssues] or [on GitLab][GitLabIssues] with a question.
+
+- If you want to add support for a hosting provider which is not yet supported:
+
+  1. Before making code changes, please create an [issue on GitHub][GitHubIssues]
+     or [on GitLab][GitLabIssues] to track the ongoing work.
```

Comment:
Is it possible that "closes #123" in a message of a commit pushed to master will close the issue with that number in both GitHub and GitLab?

Reply:
Yes, but I won't be using just the number `#123`. I'm already using full URLs, GitHub understands them: [example](https://github.com/rybak/copy-commit-reference-userscript/commit/c938c24853076ca4b3ed602a709cd9914ca7efff.patch).